readme.txt

```diff
@@ -193,9 +193,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [TBD] TBD =
 
-* Enhancement - Add notice about the availability of Paystack for Tickets Commerce. [ET-1764]
-* Enhancement - Improve performance in admin due to unnecessary Tickets Commerce calls being made. [ET-1736]
+* Enhancement - Add notice about the availability of Paystack for Tickets Commerce [ET-1764]
+* Enhancement - Improve performance in admin due to unnecessary Tickets Commerce calls being made [ET-1736]
```

Comment:
Are these changes intentional?

Reply:
Thanks for catching that. There was a merge I had to do and I must of selected the old readme entries.